scss/typography/_helpers.scss

```diff
@@ -75,4 +108,15 @@ $stat-font-size: 2.5rem !default;
     margin-#{$global-left}: 0;
     list-style: none;
   }
+  
+  // Use to create an initial aka dropcap
+  .dropcaps{
+    line-height: 1.5; // Disabling default line height of an element
```

Comment:
@ncoden i did that to let it look similar on proportion to rest of the text ... no matter what the element is

currently by default its, 
h1 to h6 => 1.4
body => 1.5
p => 1.6

But if you dont want.... no issue i will remove it @ncoden